Extensibility.md

```diff
@@ -0,0 +1,66 @@
+**Future Extensibility** 
+
+The current solution focuses mainly on post-call audio and text
```

Comment:
Everywhere we use the term "solution" we should use "accelerator" right? This is not a full solution but just a an accelerator code sample. I know Jess and team want to be very clear on the distinction in this terminology.

Reply:
to stay on the safe side i updated everything to accelerator.